manubot/cite/util.py

```diff
@@ -218,101 +218,106 @@ def is_valid_citation(
     if source not in sources:
         if source.lower() in sources:
             logging.error(
-                f'Citation sources should be all lowercase.\n'
-                f'Should {citation} use "{source.lower()}" rather than "{source}"?'
+                f'citekey sources should be all lowercase.\n'
+                f'Should {citekey} use "{source.lower()}" rather than "{source}"?'
             )
         else:
             logging.error(
-                f'Invalid citation: {citation}\n'
-                f'Source "{source}" is not valid.\n'
+                f'invalid citekey: {citekey!r}\n'
+                f'Source {source!r} is not valid.\n'
                 f'Valid citation sources are {{{", ".join(sorted(sources))}}}'
             )
         return False
 
-    inspection = inspect_citation_identifier(citation)
+    inspection = inspect_citekey(citekey)
     if inspection:
-        logging.error(f'invalid {source} citation: {citation}\n{inspection}')
+        logging.error(f'invalid {source} citekey: {citekey}\n{inspection}')
         return False
 
     return True
 
 
-def get_citation_short_id(standard_id):
+def shorten_citekey(citekey):
```

Comment:
Would it be more explicit to rename `citekey` as `standard_citekey`?

Reply:
Done in https://github.com/manubot/manubot/pull/129/commits/cf760acc177e0638ec11379af3ff18474c3c3676